index.rst

```diff
@@ -13,13 +13,93 @@
 Abstract
 ========
 
-This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
+This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. 
+The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
 
-Add content here
-================
+Guidelines for Operations
+=========================
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Survey mode is intended to be the primary mode for Auxiliary Telescope (AuxTel) operations. 
+It is a passive mode of operations, with nighttime observations driven by the Scheduler with minimal staff intervention. 
+The central tenant of operations in this mode is that the AuxTel operations are secondary to any responsibilities the summit staff may have related to Simonyi Survey Telescope operations. 
+
+This results in the following guidelines for operations in survey mode: 
+
+- The observing staff are not be required to actively monitor or report on incoming data quality.
+- Logging will be minimal; limited to short narrative statements needed to report on changes during the night or provide context for faults and recovery attempts.
+- Faults will be recovered when observing staff have available time, and the AuxTel may remain in Standby state during the night for extended periods of time. 
+- Remote support will be available for fault resolution on an on-call basis. 
+- Any faults which put hardware at risk will be result in closing the AuxTel early for the night.
```

Comment:
If they are seen in time.

Reply:
True, I'm not even sure if this line is needed. I mostly just wanted to give people more authority to make the call to close if needed, so I merged it with the next line.